packages/wonder-blocks-testing/src/harness/adapters/adapters.js

```diff
@@ -25,14 +25,9 @@ export const DefaultAdapters = {
 /**
  * The default configurations to use with the `DefaultAdapters`.
  */
-export const DefaultConfigs: Configs<typeof DefaultAdapters> = {
+export const DefaultConfigs: TestHarnessConfigs<typeof DefaultAdapters> = {
     css: css.defaultConfig,
     data: data.defaultConfig,
     portal: portal.defaultConfig,
     router: router.defaultConfig,
 };
-
-export {css};
-export {data};
-export {portal};
-export {router};
```

Comment:
No point in doing this. It's equivalent information to what gets exported above, just organized differently.